tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/Logging/ILogger/DirectSubmission/ExtendedLoggerFactoryConstructorIntegration_930_940.cs

```diff
@@ -0,0 +1,38 @@
+// <copyright file="ExtendedLoggerFactoryConstructorIntegration_930_940.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+#nullable enable
+using System;
+using System.ComponentModel;
+using Datadog.Trace.ClrProfiler.CallTarget;
+using Datadog.Trace.Configuration;
+using Datadog.Trace.DuckTyping;
+
+namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.Logging.ILogger.DirectSubmission;
+
+/// <summary>
+/// ExtendedLoggerFactory calltarget instrumentation for direct log submission
+/// </summary>
+[InstrumentMethod(
+    AssemblyName = "Microsoft.Extensions.Telemetry",
+    TypeName = "Microsoft.Extensions.Logging.ExtendedLoggerFactory",
+    MethodName = ".ctor",
+    ReturnTypeName = ClrNames.Void,
+    ParameterTypeNames = ["System.Collections.Generic.IEnumerable`1[Microsoft.Extensions.Logging.ILoggerProvider]", "System.Collections.Generic.IEnumerable`1[Microsoft.Extensions.Diagnostics.Enrichment.ILogEnricher]", "System.Collections.Generic.IEnumerable`1[Microsoft.Extensions.Diagnostics.Enrichment.IStaticLogEnricher]", "Microsoft.Extensions.Options.IOptionsMonitor`1[Microsoft.Extensions.Logging.LoggerFilterOptions]", "Microsoft.Extensions.Logging.LoggingSampler", "Microsoft.Extensions.Options.IOptions`1[Microsoft.Extensions.Logging.LoggerFactoryOptions]", "Microsoft.Extensions.Logging.IExternalScopeProvider", "Microsoft.Extensions.Options.IOptionsMonitor`1[Microsoft.Extensions.Logging.LoggerEnrichmentOptions]", "Microsoft.Extensions.Options.IOptionsMonitor`1[Microsoft.Extensions.Logging.LoggerRedactionOptions]", "Microsoft.Extensions.Compliance.Redaction.IRedactorProvider"],
+    MinimumVersion = "9.3.0",
+    MaximumVersion = "9.5.0",
```

Comment:
I think we should have an open-ended check here. It looks like this version is _also_ used in 9.5.0 in <.NET 9.0: https://github.com/dotnet/extensions/blob/main/src/Libraries/Microsoft.Extensions.Telemetry/Logging/ExtendedLoggerFactory.cs#L50-L54

Which feels like a _horrible_ design, but OK Microsoft 😅 Either way I think we need to keep this one.


```suggestion
    MaximumVersion = SupportedVersions.LatestDotNet,
``

Reply:
Oh good catch

Reply:
Definitely something weird going on in the CI failure too, .NET 9 only failed:

```
[ERR] CheckBuildLogsForErr: 02:31:55 [Error] Exception occurred when calling the CallTarget integration continuation. System.TypeInitializationException: The type initializer for 'Datadog.Trace.ClrProfiler.CallTarget.Handlers.BeginMethodSlowHandler`2' threw an exception.
 ---> Datadog.Trace.ClrProfiler.CallTarget.CallTargetInvokerException: The type or method has 12 generic parameter(s), but 11 generic argument(s) were provided. A generic argument must be provided for each generic parameter.
 ---> System.ArgumentException: The type or method has 12 generic parameter(s), but 11 generic argument(s) were provided. A generic argument must be provided for each generic
```